### PR TITLE
Python3 support for cdis_oauth2client

### PR DESCRIPTION
**Pull request by itsJiaqi**
## Details
* ran 2to3
* added some unit tests because pytest was failing because there were 0 tests

## Breaking Changes
Uses python3

### CODE REVIEW

**Comment by rudyardrichter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- tests/test_blueprint.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/uc-cdis/cdis_oauth2client/pullRequest?prid=4479387&bid=15116932)

File: cdis_oauth2client/client.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 Defines the OAuth2Client class for use by CDIS internal microservices.
 """
 
-import urllib
-import urlparse
+import urllib.request, urllib.parse, urllib.error
+import urllib.parse
```
Comment (rudyardrichter) [inline cdis_oauth2client/client.py]: Codacy Issue found: [Reimport 'urllib.parse' (imported line 7)](https://app.codacy.com/app/uc-cdis/cdis_oauth2client/pullRequest?prid=4479387)

**Comment by rudyardrichter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/test_blueprint.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/uc-cdis/cdis_oauth2client/pullRequest?prid=4479387&bid=15116932)

File: Pipfile [hunk 2]
```diff
@@ -4,6 +4,12 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+pytest = "*"
 
 [packages]
 cdis-oauth2client = {editable = true,path = "."}
+cdispyutils = "==1.0.3"
+flask = "*"
```
Comment (fantix) [inline Pipfile]: `cdispyutils` and `flask` are found in `setup.py`, they are not needed here

**Review by fantix — APPROVED** [review]
The rest is good for me

**Comment by rudyardrichter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- tests/test_blueprint.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/uc-cdis/cdis_oauth2client/pullRequest?prid=4479387&bid=15116932)

**Review by paulineribeyre — APPROVED** [review]
lgtm

**Pull request merged**